tests/integration/test_app.py

```diff
@@ -107,7 +108,11 @@ def _request(payload: dict, webhook_secret: Optional[str]) -> requests.Response:
 
 
 def test_receive_webhook(
-    webhook_logs_dir: Path, process_count: int, webhook_secret: Optional[str]
+    webhook_logs_dir: Path,
+    process_count: int,
+    webhook_secret: Optional[str],
+    # app argument is not used here but necessary for the fixture to run
+    app: Flask,  # pylint: disable = unused-argument
```

Comment:
There is an article here that should help you avoid needing to disable the rule: https://jdkandersson.com/2023/08/18/how-to-avoid-pylint-disableunused-argument-in-pytest/

Reply:
Thanks!